### PR TITLE
data:tobySolutions

### PR DESCRIPTION
**Pull request by tobySolutions**
## Check List (Check all the applicable boxes) 

- [x] My code follows the code style of this project.
- [x] My change requires changes to the documentation.
- [x] I have updated the documentation accordingly.
- [x] All new and existing tests passed.
- [x] This PR does not contain plagiarized content.
- [x] The title of my pull request is a short description of the requested changes.

## Note to reviewers

I added more content to my Milestones page. Please help merge. Thanks maintainers!

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/5682"></a>

### CODE REVIEW

File: data/tobySolutions.json [hunk 1]
```diff
@@ -134,6 +134,14 @@
       "color": "blue",
       "description": "I attended the Microsoft Learnathon organized by the ADC ecosystem and hosted at the Microsoft Nigeria's branch",
       "url": "https://twitter.com/toby_solutions/status/1623026397055680512"
+    },
+    {
+      "title": "Became Web team volunteer PAADC Cohort 7.0",
+      "date": "2023",
+      "icon": "CgCommunity",
```
Comment (eddiejaoude) [inline data/tobySolutions.json:141]: I don't think that is a valid icon, it won't block the PR but will default to another icon

**Review by eddiejaoude — APPROVED** [review]
Looks good 👍

File: data/tobySolutions.json (hunk 1 above)
Comment (eddiejaoude) [reply data/tobySolutions.json:141]: You can search for available icons here http://linkfree.io/icons

Comment (tobySolutions) [reply data/tobySolutions.json:141]: Thank you Eddie!

**Pull request merged**